be/src/runtime/descriptors.h

```diff
@@ -18,12 +18,12 @@
 #ifndef DORIS_BE_RUNTIME_DESCRIPTORS_H
 #define DORIS_BE_RUNTIME_DESCRIPTORS_H
 
+#include <ostream>
+#include <unordered_map>
 #include <vector>
-#include <tr1/unordered_map>
 #include <vector>
-#include <boost/scoped_ptr.hpp>
-#include <ostream>
 
+#include <boost/scoped_ptr.hpp>
```

Comment:
this can be replaced with std::unique_ptr too

Reply:
It's unused, I removed it.